components/sup/src/templating/helpers.rs

```diff
@@ -14,16 +14,49 @@
 
 use std::str::FromStr;
 use std::string::ToString;
+use std::collections::{BTreeMap, HashMap};
 
 use hcore::package::{PackageIdent, Identifiable};
 use hcore::fs;
-use manager::service::config::ServiceConfig;
-use handlebars::{Handlebars, Helper, RenderContext, RenderError};
+use manager::service::config::{ServiceConfig, Svc};
+use handlebars::{Handlebars, Helper, Renderable, RenderContext, RenderError, Context};
 use serde_json;
 use toml;
 
+
 type RenderResult = Result<(), RenderError>;
 
+pub fn each_alive(h: &Helper, r: &Handlebars, rc: &mut RenderContext) -> RenderResult {
+    let value = try!(h.param(0)
+            .ok_or_else(|| RenderError::new("Param not found for helper \"eachAlive\"")))
+        .value();
+
+    let template = try!(h.template()
+        .ok_or_else(|| RenderError::new("No content to render inside the helper \"eachAlive\"")));
+    let default_array = Vec::default();
+    let value_array = value.as_array().unwrap_or(&default_array);
```

Comment:
this is a total nit but an `if let` on `Ok` could avoid the need for `default_array`